tests/test_expense_tools.py

```
@@ -139,370 +108,218 @@ def test_list_expenses_with_filters(self, mock_api_request):
+    async def test_list_expenses_with_date_objects(self, mock_api_request):
         """Test listing expenses with date objects."""
-        # Set up the mock
-        mock_api_request.return_value = self.mock_expenses_list
+        mock_api_request.return_value = MOCK_EXPENSES_LIST
         
-        # Call the function with date objects
         start_date = date(2025, 1, 1)
         end_date = date(2025, 1, 31)
-        result = list_expenses(
+        
+        result = await list_expenses(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)